docs/documentation/development/bindings/xml-reference.md

```diff
@@ -83,6 +84,7 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>parameter.required</td><td>Specifies whether the value is required (optional).</td></tr>
   <tr><td>parameter.readOnly</td><td>Specifies whether the value is read-only (optional).</td></tr>
   <tr><td>parameter.multiple</td><td>Specifies whether multiple selections of options are allowed (optional).</td></tr>
+  <tr><td>parameter.unit</td><td>Specifies the unit of measurements. The unit must only be set if the type of the parameter is either integer or decimal (optional)).</td></tr>
```

Comment:
Please provide a reference to valid options (either to https://github.com/unitsofmeasurement/unit-ri/blob/master/src/main/java/tec/units/ri/unit/Units.java or pointing out that the currently valid set is shown above in the parameter definition).